 db/constants.conf | 45 ++++++++++++++++++++++++---------------------
 1 file changed, 24 insertions(+), 21 deletions(-)

diff --git a/db/constants.conf b/db/constants.conf
index 598bc89c4..fd95733e6 100644
--- a/db/constants.conf
+++ b/db/constants.conf
@@ -225,6 +225,8 @@ constants_db: {
 	EAJL_2_1:                  0x0100
 	EAJL_2_2:                  0x0200
 	EAJL_2:                    0x0300
+	EAJL_1:                    0x0400
+	
 	EAJL_UPPER:                0x1000
 	EAJL_BABY:                 0x2000
 	EAJL_THIRD:                0x4000
@@ -234,19 +236,20 @@ constants_db: {
 	EAJ_THIRDMASK:             0x4fff
 
 	EAJ_NOVICE:                0x0000
-	EAJ_SWORDMAN:              0x0001
-	EAJ_MAGE:                  0x0002
-	EAJ_ARCHER:                0x0003
-	EAJ_ACOLYTE:               0x0004
-	EAJ_MERCHANT:              0x0005
-	EAJ_THIEF:                 0x0006
-	EAJ_TAEKWON:               0x0007
-	EAJ_GUNSLINGER:            0x0009
-	EAJ_NINJA:                 0x000A
-	EAJ_GANGSI:                0x000D
 	EAJ_SUMMONER:              0x000E
 
-	EAJ_SUPER_NOVICE:          0x0100
+	EAJ_SUPER_NOVICE:          0x0400
+	EAJ_SWORDMAN:              0x0401
+	EAJ_MAGE:                  0x0402
+	EAJ_ARCHER:                0x0403
+	EAJ_ACOLYTE:               0x0404
+	EAJ_MERCHANT:              0x0405
+	EAJ_THIEF:                 0x0406
+	EAJ_TAEKWON:               0x0407
+	EAJ_GUNSLINGER:            0x0409
+	EAJ_NINJA:                 0x040A
+	EAJ_GANGSI:                0x040D
+
 	EAJ_KNIGHT:                0x0101
 	EAJ_WIZARD:                0x0102
 	EAJ_HUNTER:                0x0103
@@ -258,6 +261,7 @@ constants_db: {
 	EAJ_KAGEROUOBORO:          0x010A
 	EAJ_DEATH_KNIGHT:          0x010D
 
+	EAJ_SUPER_NOVICE_E:        0x0200
 	EAJ_CRUSADER:              0x0201
 	EAJ_SAGE:                  0x0202
 	EAJ_BARDDANCER:            0x0203
@@ -290,14 +294,15 @@ constants_db: {
 	EAJ_STALKER:               0x1206
 
 	EAJ_BABY:                  0x2000
-	EAJ_BABY_SWORDMAN:         0x2001
-	EAJ_BABY_MAGE:             0x2002
-	EAJ_BABY_ARCHER:           0x2003
-	EAJ_BABY_ACOLYTE:          0x2004
-	EAJ_BABY_MERCHANT:         0x2005
-	EAJ_BABY_THIEF:            0x2006
-
-	EAJ_SUPER_BABY:            0x2100
+	EAJ_SUPER_BABY:            0x2400
+	EAJ_BABY_SWORDMAN:         0x2401
+	EAJ_BABY_MAGE:             0x2402
+	EAJ_BABY_ARCHER:           0x2403
+	EAJ_BABY_ACOLYTE:          0x2404
+	EAJ_BABY_MERCHANT:         0x2405
+	EAJ_BABY_THIEF:            0x2406
+
+	EAJ_SUPER_BABY_E:          0x2100
 	EAJ_BABY_KNIGHT:           0x2101
 	EAJ_BABY_WIZARD:           0x2102
 	EAJ_BABY_HUNTER:           0x2103
@@ -312,7 +317,6 @@ constants_db: {
 	EAJ_BABY_ALCHEMIST:        0x2205
 	EAJ_BABY_ROGUE:            0x2206
 
-	EAJ_SUPER_NOVICE_E:        0x4100
 	EAJ_RUNE_KNIGHT:           0x4101
 	EAJ_WARLOCK:               0x4102
 	EAJ_RANGER:                0x4103
@@ -341,7 +345,6 @@ constants_db: {
 	EAJ_GENETIC_T:             0x5205
 	EAJ_SHADOW_CHASER_T:       0x5206
 
-	EAJ_SUPER_BABY_E:          0x6100
 	EAJ_BABY_RUNE:             0x6101
 	EAJ_BABY_WARLOCK:          0x6102
 	EAJ_BABY_RANGER:           0x6103
